2025-05-21 18:55:07,348 [ERROR] Tkinter callback exception
Traceback (most recent call last):
  File "C:\Users\chris\AppData\Local\Programs\Python\Python313\Lib\tkinter\__init__.py", line 2068, in __call__
    return self.func(*args)
           ~~~~~~~~~^^^^^^^
  File "c:\Users\chris\Documents\energy_neural_system\main.py", line 1130, in <lambda>
    window_menu.add_command(label="Change Sensory Resolution", command=lambda: change_sensory_resolution(drawwin, system_ref, capture_ref))
                                                                               ^^^^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'change_sensory_resolution' is not defined
